segmenti/segmenti.py

```diff
@@ -26,47 +28,47 @@ def sostenitori_cri(queryset):
 
 
 def aspiranti_volontari_iscritti_ad_un_corso(queryset):
-    pass
+    return queryset.filter(aspirante__isnull=False, partecipazioni_corsi__isnull=False)
 
 
 def tutti_i_presidenti(queryset):
-    pass
+    return queryset.filter(delega__tipo=PRESIDENTE)
 
 
 def presidenti_comitati_locali(queryset):
-    pass
+    return tutti_i_presidenti(queryset).filter(appartenenze__sede__tipo=Sede.COMITATO, appartenenze__sede__estensione=LOCALE)
```

Comment:
Le appartenenze (volontario, sostenitore, ...) sono da considerare separate rispetto alle deleghe (presidente sede, referente attivita, ...). In alcuni casi, un volontario presso un Comitato puo' essere presidente presso un altro.

Il modo corretto per trovare i Comitati legali sarebbe seguendo la relazione `deleghe` (da `anagrafica.models.Delega`), piuttosto che `appartenenze`. Notate pero' che `Delega` e' correlata a `oggetto`, che puo' essere una Sede. Questa e' pero una relazione generica (tramite `oggetto_tipo`, che deve corrispondere al ContentType corrispondente alla Sede, e `oggetto_id`).

Se preferite possiamo sentirci telefonicamente/hangout in merito